src/Form/Extension/HelpMessageTypeExtension.php

```diff
@@ -29,4 +29,9 @@ public function buildView(FormView $view, FormInterface $form, array $options):
     {
         $view->vars['help'] = $options['help'] ?? '';
     }
+
+    public static function getExtendedTypes(): iterable
```

Comment:
Is `iterable` supported in PHP 7.1? If not, you can't have this return type unless you change the php requirement in composer.

Reply:
It is actually supported.

Reply:
Ok, thanks for the confirmation.